driver/test_kubeflow_workloads.py

```diff
@@ -16,7 +16,9 @@
 JOB_TEMPLATE_FILE = ASSETS_DIR / "test-job.yaml.j2"
 PROFILE_TEMPLATE_FILE = ASSETS_DIR / "test-profile.yaml.j2"
 
-TESTS_DIR = os.path.abspath(Path("tests"))
+TESTS_LOCAL_RUN = os.environ.get("LOCAL").replace('"', "")
```

Comment:
you can use python's `eval` to evaluate `TESTS_LOCAL_RUN`  to a boolean type

Reply:
and in the `test-job.yaml.j2` write the `if` expression to be `if tests_local_run`

Reply:
Pushed a commit for this.